test_aodndata/moorings/test_mooringsProductsHandler.py

```diff
@@ -0,0 +1,37 @@
+import httpretty
+import os
+import re
+import unittest
+
+from aodncore.pipeline import PipelineFilePublishType, PipelineFileCheckType, FileType
+from aodncore.pipeline.exceptions import ComplianceCheckFailedError, InvalidFileContentError, InvalidFileNameError
+from aodncore.testlib import HandlerTestCase, make_zip
+from aodndata.moorings.products_handler import MooringsProductsHandler
+
+TEST_ROOT = os.path.dirname(__file__)
+GOOD_MANIFEST = os.path.join(TEST_ROOT, 'product_manifest.json')
+
+GETCAPABILITIES_FILE = os.path.join(TEST_ROOT, 'getCapabilities.xml')
+
+with open(GETCAPABILITIES_FILE) as f:
+    TEST_GETCAPABILITIES_RESPONSE = httpretty.Response(f.read())
+
+TEST_GETFEATURE_RESPONSE = httpretty.Response(
```

Comment:
I think we should just put all these responses into separate files (despite me having inlined them in the first place!)

Reply:
Done